### ISSUE TITLE
Superset Welcome page CSS is not formatting correctly [macOS]

### ISSUE THREAD

**Issue by DavidM0808** [apache/superset#25682]
This issue is produced by running superset using Docker Desktop on Mac.

Additionally, `superset_init` runs without any issue, it's just that the welcome page is in the wrong format.

#### How to reproduce the bug

1. Go to `terminal`
2. Navigate to the `superset` directory
3. Type in `docker compose up`
4. Wait until `superset_init` container exits with code `0`
5. See error

### Expected results

After logging in, a nicely formatted welcome page is displayed.

### Actual results

A very poorly formatted welcome page that's mostly in Times New Roman Font and bullet points.

#### Screenshots

<img width="1728" alt="Screenshot 2023-10-17 at 3 38 09 PM" src="https://github.com/apache/superset/assets/84249204/b2c8bb38-9527-4e4b-a2cb-0f42d4c48f15">

### Environment

- browser type and version: `chrome Version 118.0.5993.70`
- superset version: `0.0.0-dev` (As of Oct 18, 2023)
- python version: `python --3.9.3`
- node.js version: `node 16`
- any feature flags active: No

### Checklist

- [x] I have checked the superset logs for python stacktraces and included it here as text if there are any.
- [x] I have reproduced the issue with at least the latest released version of superset.
- [x] I have checked the issue tracker for the same issue and I haven't found one similar.

### Additional context

Here's a portion of the log from `superset_node` close to 100% of webpack extraction:

```
2023-10-17 15:51:49 <s> [webpack.Progress] 95% emitting emit
2023-10-17 15:51:49 <s> [webpack.Progress] 95% emitting emit
2023-10-17 15:51:49 <s> [webpack.Progress] 98% emitting after emit
2023-10-17 15:51:49 <s> [webpack.Progress] 98% emitting after emit SizeLimitsPlugin
2023-10-17 15:51:49 <s> [webpack.Progress] 98% emitting after emit
2023-10-17 15:51:49 <s> [webpack.Progress] 99% done plugins
2023-10-17 15:51:49 <s> [webpack.Progress] 99% done plugins
2023-10-17 15:51:49 <s> [webpack.Progress] 99% 
2023-10-17 15:51:49 
2023-10-17 15:51:52 849 assets
2023-10-17 15:51:52 12019 modules
2023-10-17 15:51:52 
2023-10-17 15:51:52 WARNING in ./node_modules/@data-ui/shared/esm/enhancer/WithTooltip.js 3:60-78
2023-10-17 15:51:52 export 'withTooltipPropTypes' (imported as 'vxTooltipPropTypes') was not found in '@vx/tooltip/build/tooltips/TooltipWithBounds' (possible exports: __esModule, default)
2023-10-17 15:51:52 
2023-10-17 15:51:52 WARNING in ./node_modules/moment/min/moment-with-locales.js 2096:16-50
2023-10-17 15:51:52 Module not found: Error: Can't resolve './locale' in '/app/superset-frontend/node_modules/moment/min'
2023-10-17 15:51:52 
2023-10-17 15:51:52 WARNING in ./src/utils/textUtils.ts 21:11-44
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../../../superset_text' in '/app/superset-frontend/src/utils'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./plugins/legacy-plugin-chart-world-map/src/WorldMap.js 27:0-55
2023-10-17 15:51:52 Module not found: Error: Can't resolve 'datamaps/dist/datamaps.world.min' in '/app/superset-frontend/plugins/legacy-plugin-chart-world-map/src'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/css.js 680:47-71
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/css' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/html.js 771:47-78
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/javascript' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/html.js 1290:47-71
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/css' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/html.js 2459:47-72
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/html' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/javascript.js 771:47-78
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/javascript' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/json.js 300:47-72
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/json' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/markdown.js 771:47-78
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/javascript' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/markdown.js 1436:47-71
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/xml' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/markdown.js 1955:47-71
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/css' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/brace/mode/markdown.js 2505:47-72
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../worker/html' in '/app/superset-frontend/node_modules/brace/mode'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/200.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/200.css) 8:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-greek-ext-200-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/200.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/200.css) 10:36-105
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-vietnamese-200-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/200.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/200.css) 11:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-ext-200-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/200.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/200.css) 12:36-100
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-200-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/400.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/400.css) 8:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-greek-ext-400-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/400.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/400.css) 10:36-105
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-vietnamese-400-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/400.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/400.css) 11:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-ext-400-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/400.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/400.css) 12:36-100
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-400-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/500.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/500.css) 8:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-greek-ext-500-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/500.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/500.css) 10:36-105
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-vietnamese-500-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/500.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/500.css) 11:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-ext-500-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/500.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/500.css) 12:36-100
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-500-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/600.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/600.css) 8:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-greek-ext-600-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/600.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/600.css) 10:36-105
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-vietnamese-600-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/600.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/600.css) 11:36-104
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-ext-600-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/@fontsource/inter/600.css (./node_modules/css-loader/dist/cjs.js??ruleSet[1].rules[5].use[1]!./node_modules/@fontsource/inter/600.css) 12:36-100
2023-10-17 15:51:52 Module not found: Error: Can't resolve './files/inter-latin-600-normal.woff2' in '/app/superset-frontend/node_modules/@fontsource/inter'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array/fill.js 1:0-39
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../../modules/es6.array.fill' in '/app/superset-frontend/node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array/fill.js 2:0-58
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../../modules/_core' in '/app/superset-frontend/node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array/includes.js 1:0-43
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../../modules/es7.array.includes' in '/app/superset-frontend/node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array/includes.js 2:0-62
2023-10-17 15:51:52 Module not found: Error: Can't resolve '../../modules/_core' in '/app/superset-frontend/node_modules/react-jsonschema-form/node_modules/core-js/library/fn/array'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./src/components/AsyncAceEditor/index.tsx 32:21-46
2023-10-17 15:51:52 Module not found: Error: Can't resolve 'brace/mode/yaml' in '/app/superset-frontend/src/components/AsyncAceEditor'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./src/components/AsyncAceEditor/index.tsx 35:26-56
2023-10-17 15:51:52 Module not found: Error: Can't resolve 'brace/theme/textmate' in '/app/superset-frontend/src/components/AsyncAceEditor'
2023-10-17 15:51:52 
2023-10-17 15:51:52 ERROR in ./src/components/AsyncAceEditor/index.tsx 36:24-52
2023-10-17 15:51:52 Module not found: Error: Can't resolve 'brace/theme/github' in '/app/superset-frontend/src/components/AsyncAceEditor'
2023-10-17 15:51:52 
2023-10-17 15:51:52 webpack 5.88.1 compiled with 34 errors and 3 warnings in 280852 ms
2023-10-17 15:51:52 <s> [webpack.Progress] 99% cache store build dependencies
2023-10-17 15:51:52 <s> [webpack.Progress] 99% cache store build dependencies
2023-10-17 15:51:52 <s> [webpack.Progress] 99% cache begin idle
2023-10-17 15:51:52 <s> [webpack.Progress] 99% cache begin idle
2023-10-17 15:51:52 <s> [webpack.Progress] 100% 
```

**Comment by DavidM0808** [comment]
Update to the issue:

I have also tried running `docker compose down -v`, then run `docker compose up` again, the issue mentioned above still exists.

**Comment by sfirke** [comment]
What version of Superset are you running?  Seems like a possible duplicate of https://github.com/apache/superset/issues/25448 which was recently fixed.

**Comment by sfirke** [comment]
Sorry I see you are running 2.1.2.  Perhaps that's the problem.  Thanks for reporting.  Try changing the content security policy via your config, as implemented in the PR that fixed this in the other issue: https://github.com/apache/superset/pull/25468

**Comment by DavidM0808** [comment]
@sfirke do you know where I can find the CSP? is it in the `superset_config.py` that's in the `docker` folder or just `config.py`?

**Comment by DavidM0808** [comment]
Sounds good, let me try getting a stable version and see how it goes!

**Comment by DavidM0808** [comment]
The log from `superset_init` is shown as follows:

```
superset_init          | sqlalchemy.exc.OperationalError: (psycopg2.errors.DeadlockDetected) deadlock detected
superset_init          | DETAIL:  Process 80 waits for AccessExclusiveLock on relation 16404 of database 16384; blocked by process 84.
superset_init          | Process 84 waits for AccessShareLock on relation 16412 of database 16384; blocked by process 80.
superset_init          | HINT:  See server log for query details.
superset_init          | 
superset_init          | [SQL: ALTER TABLE dashboard_roles DROP CONSTRAINT dashboard_roles_role_id_fkey]
superset_init          | (Background on this error at: https://sqlalche.me/e/14/e3q8)
superset_init exited with code 1
```

**Comment by rusackas** [comment]
This is very odd. I use Docker desktop on Mac with vanilla installations of Superset pretty frequently, and haven't had problems with these fonts (or other assets) loading. We're no longer supporting Superset 2.x, but I'm curious if you've managed to get things up and running with official releases of 3.0.x, 3.1.x, or 4.0.

**Comment by rusackas** [comment]
Guessing by the silence that this has been somehow resolved, or OP has moved along... closing, but happy to reopen if warranted.
